crates/rolldown/src/module_finalizers/scope_hoisting/mod.rs

```diff
@@ -95,12 +95,6 @@ impl<'me, 'ast> ScopeHoistingFinalizer<'me, 'ast> {
         // `import_foo`
         let binding_name_for_wrapper_call_ret = self.canonical_name_for(rec.namespace_ref);
 
-        // If the module is an ESM module that follows the Node.js ESM spec, such as
-        // - extension is `.mjs`
-        // - `package.json` has `"type": "module"`
-        // , we need to consider to stimulate the Node.js ESM behavior for maximum compatibility.
-        let should_consider_node_esm_spec = self.ctx.module.ecma_view.def_format.is_esm();
```

Comment:
Could you keep this part of code if it doesn't have bugs? It makes understanding that why we should add a `1` in the `toESM` call easier.

Other changes are fine.

Reply:
I moved the comment to `crates/rolldown_common/src/module/normal_module.rs`.

Reply:
Updated.